controllers/kogitobuild_controller.go

```diff
@@ -77,31 +60,31 @@ type KogitoBuildReconciler struct {
 // +kubebuilder:rbac:groups=sources.knative.dev,resources=sinkbindings,verbs=get;list;watch;create;delete;update
 // +kubebuilder:rbac:groups=integreatly.org,resources=grafanadashboards,verbs=get;create;list;watch;create;delete;update
 func (r *KogitoBuildReconciler) Reconcile(req ctrl.Request) (result ctrl.Result, resultErr error) {
-	r.Log.Info("Reconciling for", "KogitoBuild", req.Name, "Namespace", req.Namespace)
+	log := r.Log.WithValues("name", req.Name, "namespace", req.Namespace)
+	log.Info("Reconciling for KogitoBuild")
 
 	// fetch the requested instance
-	instance := &appv1beta1.KogitoBuild{}
-	exists, err := kubernetes.ResourceC(r.Client).FetchWithKey(req.NamespacedName, instance)
-	if err != nil {
-		return reconcile.Result{Requeue: false}, err
-	}
-	if !exists {
-		return reconcile.Result{Requeue: false}, nil
+	buildInstanceHandler := internal.NewKogitoBuildHandler(r.Client, log)
+	instance, resultErr := buildInstanceHandler.FetchKogitoBuildInstance(req.NamespacedName)
+	if resultErr != nil {
+		return
+	} else if instance == nil {
+		log.Info("Kogito Build not found")
```

Comment:
I would expect the return statement here.

Reply:
Thanks for pointed it out. I missed that return statement.

Reply:
plus set the log level to warn?